uktrade/data-hub-frontend
# Switch over to the new React events collection page

## Description of change

Switches out the old Nunjucks implementation of the events collection page for the React version.

## Test instructions

Visit `/events`

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
